### DISCUSSION TITLE
CSV won't export

### DISCUSSION THREAD

**Discussion by fidomullis** [pwillworth/dfkreport#31] [General]
Hi pwillworth,
Lila won't export CSV. It's for a period of one month (June 2023). The member's page says 134063 txns. I've regenerated it twice. Both times after pressing 'Export CSV', and some wheel spinning, nothing happens. Exporting to koinly Univirsal, Transactions tab.
Thanks

**Comment by pwillworth** [comment]
Hey thanks for letting me know.  I'm going to take a look at this and get back to you.  Just need to work on re-creating it so I can see what's happening.

**Comment by pwillworth** [comment]
@fidomullis I'm still looking into this, but if you just need the csv, you should be able to get it via direct link (worked for me)
You just have to replace the ID (32 random characters on end of your report url) in the link below and put it in your browser.

https://dfkreport.cognifact.com/view?formatType=csv&contentType=transaction&contentFile=PUTYOURREPORTIDHERE&csvFormat=koinly

It will spin for a minute or two while it downloads the several megabytes of data, then popup the file.

I'm still working on why it doesn't load from report page if you want to just wait on that.

**Comment by pwillworth** [comment]
@fidomullis Go ahead and give that another shot now.  There was an issue with the criteria that was letting it try to export quest data with a date range greater than 1 week, which made it fail.  So, it should work now, but keep in mind questing transactions will not be included unless you do a different report with 1 week range.

The quest data limitation had to be added a couple months ago because the platform was having trouble handling huge amounts of data in the report.  I'm in progress on a re-vamp which should improve this to some degree so that limitation should be loosed up some soon.

**Comment by fidomullis** [comment]
Oh wowser! I've got a few wallets. This means I'll have to re-export with the 1-week intervals. The quests are kinda the main component. When you say 'a couple of months ago', is there a specific date?
And thanks for looking into this. Once again, Lila's Ledger is the best crypto tax tool for DFK out there.

**Reply by pwillworth** [reply]
I think it was in the last week of July when you would have started seeing quest data limited by date range on the report display page.  To be clear though, that wasn't the case until yesterday for CSV export, so anything you were able to sucessfully export before yesterday should have quest data included.

If you have multiple wallets, I would suggest creating a wallet group so can just do them all at the same time, unless all of them combined is so much data that it breaks the export again.  Hopefully not, and once I get the update done that I'm working on now, larger data exports should be doable again.

**Reply by testdfkguy** [reply]
I was also having an issue exporting before but this seems to have fixed it for me as well! Unfortunately I'm also in the boat of having a massive number of quest tx transactions so I'm also looking forward to the large data export update.  Thanks for all of your hard work!!

**Comment by fidomullis** [comment]
I've re-exported from Lila in weekly increments for the txns and reimported into koinly. All reimported transactions have been skipped by koinly (not added as new). This makes me think that the original export from Lila with the supposed 1-week quests limit had no 1-week limit after all. What do you think?

**Reply by pwillworth** [reply]
Anything you exported from Lila's before Sept. 11th would have not had a limit, so yeah it was probably there if you exported it before then.

**Reply by fidomullis** [reply]
OK, makes sense. Thanks.

**Comment by fidomullis** [comment]
For clarity, I didn't delete the original imports from koinly.

**Comment by pwillworth** [comment]
Hi all, today I released an update that includes a change to how the CSV export works.  It should allow for exporting more than a week now.

**Comment by fidomullis** [comment]
Thanks very much!
